lib/views/create_project_page.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:client_backoffice/views/create_project_form.dart';
+import 'package:client_common/views/simple_page.dart';
+import 'package:flutter/material.dart';
+import 'package:lenra_components/lenra_components.dart';
+
+class CreateProjectPage extends StatelessWidget {
```

Comment:
Same here add a bool parameter and add first when true in the message, so you can remove CreateFristProjectPage